web/images/web-py34/Dockerfile

```diff
@@ -0,0 +1,29 @@
+### AUTOGENERATED - DO NOT EDIT
+### INSTEAD EDIT THE FILE parts/Dockerfile-header
+from unit9/base:latest
+maintainer Kamil Cholewiński <kamil.cholewinski@unit9.com>
+
+workdir /app
+run adduser --system \
+    --home /app --no-create-home \
+    --disabled-login --group app \
+    --uid 999
+### AUTOGENERATED - DO NOT EDIT
+### INSTEAD EDIT THE FILE parts/Dockerfile-py34
+run apt-get update && \
+    apt-get install --yes --no-install-recommends \
+            python3-all-dev \
+            python3-pip \
+            python3-wheel \
+            uwsgi-plugin-python3 \
+    && rm -rf /var/cache/apt /var/lib/apt/lists
+run pip3 install -U pip wheel
+
+env PORT=5000
+env PYTHON_MODULE=backend
+env PYTHON_CALLABLE=app
+### AUTOGENERATED - DO NOT EDIT
+### INSTEAD EDIT THE FILE parts/Dockerfile-footer
+add run /etc/service/backend/run
```

Comment:
`add run`? is it a shorthand for `add /etc/service/backend/run` and then run it?